core/test/integration/api/api_schedules_spec.js

```diff
@@ -179,7 +179,7 @@ describe('Schedules API', function () {
             originalCannotScheduleAPostBeforeInMinutes = config.times.cannotScheduleAPostBeforeInMinutes;
 
             // we can insert published_at less then 5minutes
-            config.times.cannotScheduleAPostBeforeInMinutes = -5;
+            config.times.cannotScheduleAPostBeforeInMinutes = -15;
```

Comment:
i've tried using `configUtils`, but this utility doesn't work if a default value is defined inside `server/config/index.js` (see https://github.com/TryGhost/Ghost/pull/6861/files#diff-198522c016d444fe14bb9882f7614d2dR237). The `_.merge` is in a wrong order, it merges the default values into the custom values.